sdk-libs/macros/MACRO-NEW.md

```
@@ -0,0 +1,792 @@
+    PackedGameSession {
+        data: PackedGameSession,
+        user_idx: u8,
+        authority_idx: u8,
+    },
+
+    // CToken variant (unchanged)
+    CTokenData(CTokenData),
+}
+```
+
+### 2.2 Seeds Structs (All Seeds - ctx._ + data._)
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)